docs/index.rst

```diff
@@ -101,19 +101,22 @@ Basic usage
 
 .. testcode::
 
-   class MLP(nn.Module):
+   class MLP(nn.Module):                    # dataclass Modules
```

Comment:
Suggestion: Since all `Module`s are Python3.7 dataclasses, then

```suggestion
   class MLP(nn.Module):                    # create a Flax Module dataclass
```